.ci/dockers/ubuntu-14.04/Dockerfile

```diff
@@ -41,6 +41,26 @@ RUN curl -sL https://cmake.org/files/v3.14/cmake-3.14.1-Linux-x86_64.sh -o cmake
  && ./cmake.sh --prefix=/usr/local --exclude-subdir \
  && rm cmake.sh
 
+# Install Java
+RUN add-apt-repository ppa:openjdk-r/ppa -y \
```

Comment:
OpenJDK-8 is available from default ppa only since Ubuntu 14.10, but we use 14.04.
https://askubuntu.com/a/944260